specification/confidentialledger/resource-manager/Microsoft.ConfidentialLedger/preview/2023-06-28-preview/confidentialledger.json

```diff
@@ -0,0 +1,830 @@
+          },
+          {
+            "name": "confidentialLedger",
+            "in": "body",
+            "description": "Confidential Ledger Create Request Body",
+            "required": true,
+            "schema": {
+              "$ref": "#/definitions/ConfidentialLedger"
+            }
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "The request was successful and ledger was created; response contains the Confidential Ledger",
+            "schema": {
+              "$ref": "#/definitions/ConfidentialLedger"
+            }
+          },
+          "201": {
+            "description": "The Confidential Ledger resource will be created asynchronously. Response contains the Confidential Ledger being provisioned",
+            "schema": {
+              "$ref": "#/definitions/ConfidentialLedger"
+            }
+          },
+          "default": {
+            "description": "Error response describing the reason for operation failure. 404 - NotFound (Confidential Ledger could not be created/updated)",
+            "schema": {
+              "$ref": "../../../../../common-types/resource-management/v3/types.json#/definitions/ErrorResponse"
+            }
+          }
+        },
+        "x-ms-examples": {
+          "ConfidentialLedgerCreate": {
+            "$ref": "./examples/ConfidentialLedger_Create.json"
+          }
+        }
+      },
+      "patch": {
+        "tags": [
+          "Confidential Ledger"
+        ],
+        "summary": "Update Confidential Ledger properties",
+        "description": "Updates properties of Confidential Ledger",
+        "operationId": "Ledger_Update",
+        "x-ms-long-running-operation": true,
+        "parameters": [
+          {
+            "$ref": "../../../../../common-types/resource-management/v3/types.json#/parameters/ApiVersionParameter"
+          },
+          {
+            "$ref": "../../../../../common-types/resource-management/v3/types.json#/parameters/SubscriptionIdParameter"
+          },
+          {
+            "$ref": "../../../../../common-types/resource-management/v3/types.json#/parameters/ResourceGroupNameParameter"
+          },
+          {
+            "$ref": "#/parameters/LedgerNameParameter"
+          },
+          {
+            "name": "confidentialLedger",
+            "in": "body",
+            "description": "Confidential Ledger request body for Updating Ledger",
+            "required": true,
+            "schema": {
+              "$ref": "#/definitions/ConfidentialLedger"
+            }
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "The request was successful and ledger was recovered; response contains the Confidential Ledger",
+            "schema": {
+              "$ref": "#/definitions/ConfidentialLedger"
+            }
+          },
+          "201": {
+            "description": "The Confidential Ledger resource will be updated asynchronously"
```

Comment:
>updated [](http://example.com/codeflow?start=69&length=7)

201 should mean Created not updated

Reply:
See remarks below about 201 created is not recommended to be achievable using "patch"

Reply:
The response status for this resource has been 201 for multiple preview versions and would require a code change on our end. As I am not modifying this resource or update call in my latest preview version release, I would prefer to keep things as they are but will create a follow-up to address this and make the necessary changes for it to return a 202 for an upcoming stable version.